dashboard/app/views/levels/_external.html.haml

```diff
@@ -19,7 +19,8 @@
       - if @script.try(:old_professional_learning_course?) && @script_level
         = link_to @script_level.end_of_lesson? ? t('done_with_module') : t('next_resource'), @script_level.next_level_or_redirect_path_for_user(current_user), class: 'btn btn-large pull-right btn-primary submitButton'
       - elsif !(video && use_large_video_player)
-        %a.btn.btn-large.btn-primary.next-lesson.submitButton.pull-right= t('continue')
+        .buttons.below-content-right{style: "float: right; max-width: 20%"}
```

Comment:
Just to confirming - I'm guessing the `float:right; max-width 20%` is what's replacing the `pull-right` class?

Reply:
Yes. I think I'll go ahead and just use `pull-right` and remove the `style`. Thanks!